doc/developer/design/20230903_avro_doc.md

```diff
@@ -114,10 +114,8 @@ to the value schema; a `DOC ON ...` option applies to both types of schemas.
 Each `DOC ON` option applies to either a column or a type:
 
   * Column comments (`[KEY|VALUE] DOC ON COLUMN ...`) name a column in the
-    format `[[[db.]schema.]object.]column`. If the object name is omitted, the
-    column name is taken to be a name of a column in the top level of the
-    materialized view. Object names are looked up according to usual SQL name
-    resolution rules for the search path and active database.
+    format `[[db.]schema.]object.column`. Object names are looked up according to usual
```

Comment:
Updated the implementatoin to keep it consistent with the naming resolution for the type below `[[db.]schema.]object` and with the `COMMENT ON` sql. Also, sinks can't be queried and they don't have a relation desc.

Reply:
👍🏽